packages/commercetools/theme/components/Login.vue

```diff
@@ -0,0 +1,166 @@
+<template>
```

Comment:
How about renaming this component to `LoginModal` as it better represents what it is? I know it's a detail but since we already have `CartSidebar` its good to follow convention